### ISSUE TITLE
Try: approx G-cell activation via anisotropic soft EDT

### ISSUE THREAD

**Issue by skosch** [skosch/YinYangFit#11]
1. Compute V1C activations as usual
2. For each V1C scale/orientation, compute a soft EDT (via power-means) that is strongly anisotropic, oriented at 90° to the V1C orientation. (Challenge: do this without running into major numerical underflow issues.)
3. Compute the angle of the local gradients, and use them as relevance maps: distance fields are relevant only where the gradients are aligned with the principal direction of the EDT.
4. At each location (x, y), use the distance fields and relevance maps to compute an estimate of distance mean, variance, and strength. Optional: discount strength values wherever distances from left and right are both present (i.e. within counters).
5. Optional: use further sampling to increase confidence in the local strength values. This should be way more efficient that doing e.g. a complete Hough transform from scratch.
6. Place two letters at a distance and combine distance mean, variance, and strength values into local salience and penalty values (e.g. via a neural net, for now, trained by minimizing the salience-averaged penalties).
